packages/app/src/app/pages/Sandbox/Editor/Header/Buttons/PickButton/PickButton.tsx

```diff
@@ -1,33 +1,36 @@
 import React from 'react';
-import { observer } from 'mobx-react-lite';
-import { useSignals, useStore } from 'app/store';
+import { inject, hooksObserver } from 'app/componentConnectors';
 import { Button } from './elements';
 
-export const PickButton = observer(() => {
-  const {
-    explore: { pickSandboxModal },
-  } = useSignals();
-  const {
-    editor: {
-      currentSandbox: { id, title, description, owned },
-    },
-  } = useStore();
+export const PickButton = inject('store', 'signals')(
```

Comment:
I don't thinks if we want this back actually. 🤔
The hooks are a real improvement I think.
Using the hooks will provide us with code completion and can give use typechecking too btw.

CC @CompuIves @SaraVieira @Saeris

Reply:
@MichaelDeBoey Check out this youtube clip: https://www.youtube.com/watch?v=LNKxAVhnPZo&feature=youtu.be :) 

We have to move back to this approach before we can move to hooks. The reason is that we need the components to work with both Cerebral and Overmind. After we get that running we can use the Overmind hook instead:

```ts
const { state, actions } = useOvermind()
```

It does not require any HOC, so no observer HOC or anything :)

But yeah... using `useStore` and `useSignals` is definitely an improvement, but it was a bit premature I realize now. Have to get over to Typescript and Overmind first :)